src/data/tasks/personal-details.json

```
@@ -0,0 +1,100 @@
+{
+  "id": "personal-details",
+  "points": 2,
+  "icon": "Face",
+
+  "eng": {
+    "title": "Personal Details",
+    "description": "Earn Khetha points by telling us a bit about yourself and how you would like to receive your airtime rewards",
+    "questions": [
+      {
+        "title": "Do you give us permission to save your information?",
+        "description": "Your information will be kept confidential",
+        "format": "buttons",
+        "options": [
+          "Yes",
+          "No"
+        ]
+      },
+
+      {
+        "title": "What will your gaming name be?",
+        "description": "You can choose an alias if you want",
+        "format": "string"
+      },
+
+
+      {
+        "title": "Which network provider are you with?",
+        "description": "This information will be used for airtime prizes",
+        "format": "checkboxes",
```

From Lebo (from sheet comments): These should be exclusive (not multiselect).

```suggestion
        "format": "select",
```